src/widgets/allergies/allergies-detailed-summary.css

```diff
@@ -7,6 +7,20 @@
   padding: 0.25rem 0.5rem 0 0.5rem;
 }
 
+.allergyName {
+  font-weight: 500;
+  font-size: 1.188rem;
+}
+
+.allergySeverity {
+  font-size: 1.063rem;
+  letter-spacing: 0.063rem;
+}
+
+.allergySeverity svg {
+  margin-right: 0.375rem;
+}
```

Comment:
Something like `omrs-margin-right-4`? https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--spacing-padding-and-margin